packages/paste-core/components/side-panel/src/SidePanelHeader.tsx

```diff
@@ -0,0 +1,77 @@
+import { Box, safelySpreadBoxProps } from "@twilio-paste/box";
+import { Button } from "@twilio-paste/button";
+import { CloseIcon } from "@twilio-paste/icons/esm/CloseIcon";
+import * as React from "react";
+
+import { SidePanelContext } from "./SidePanelContext";
+import type { SidePanelHeaderProps } from "./types";
+
+type SidePanelCloseButtonProps = {
+  setIsOpen: React.Dispatch<React.SetStateAction<boolean>>;
+  i18nCloseSidePanelTitle: string;
+  sidePanelId: string;
+  isOpen: boolean;
+};
+
+const SidePanelCloseButton: React.FC<React.PropsWithChildren<SidePanelCloseButtonProps>> = ({
+  setIsOpen,
+  i18nCloseSidePanelTitle,
+  sidePanelId,
+  isOpen,
+}) => {
+  const internalButtonRef = React.useRef<HTMLButtonElement>(null);
+
+  React.useEffect(() => {
+    internalButtonRef.current?.focus({ preventScroll: true });
+  }, []);
+
+  return (
+    <Button
+      variant="secondary_icon"
+      size="reset"
+      onClick={() => {
+        setIsOpen(false);
+      }}
+      ref={internalButtonRef}
+      aria-controls={sidePanelId}
+      aria-expanded={isOpen}
+      aria-label={i18nCloseSidePanelTitle}
+    >
+      <CloseIcon decorative={false} title={i18nCloseSidePanelTitle} size="sizeIcon50" />
+    </Button>
+  );
+};
+SidePanelCloseButton.displayName = "SidePanelCloseButton";
+
+const SidePanelHeader = React.forwardRef<HTMLDivElement, SidePanelHeaderProps>(
+  ({ element = "SIDE_PANEL_HEADER", children, ...props }, ref) => {
+    const { sidePanelId, isOpen, setIsOpen, i18nCloseSidePanelTitle } = React.useContext(SidePanelContext);
+    return (
+      <Box
+        width="100%"
+        padding="space70"
+        borderBottomStyle="solid"
+        borderBottomColor="colorBorderWeaker"
+        borderBottomWidth="borderWidth10"
```

Comment:
todo: Remove bottom border from Header, and add a Separator . This is so people can add Tabs to Side Panel cleanly.